### PR TITLE
Adding error codes for mysqltoolsservice

### PR DESCRIPTION
**Pull request by rishky-msft**
Adding specific error codes instead of default zero

### CODE REVIEW

**Review by nitishsharma25 — APPROVED** [review]

File: ossdbtoolsservice/connection/connection_service.py [hunk 1]
```diff
@@ -202,10 +203,10 @@ def handle_list_databases(self, request_context: RequestContext, params: ListDat
         try:
             connection = self.get_connection(params.owner_uri, ConnectionType.DEFAULT)
         except ValueError as err:
-            request_context.send_error(str(err))
+            request_context.send_error(message=str(err), code=OssdbErrorConstants.LIST_DATABASE_VALUE_ERROR)
```
Comment (rishky-msft) [inline ossdbtoolsservice/connection/connection_service.py:206]: should be LIST_DATABASE_GET_CONNECTION_VALUE_ERROR

File: ossdbtoolsservice/query_execution/query_execution_service.py [hunk 2]
```diff
@@ -34,6 +35,7 @@
 )
 
 from ossdbtoolsservice.driver import ServerConnection
+from ossdbtoolsservice.exception.OssdbErrorConstants import OssdbErrorConstants
```
Comment (rishky-msft) [inline ossdbtoolsservice/query_execution/query_execution_service.py:38]: duplicate import

File: ossdbtoolsservice/connection/connection_service.py (hunk 1 above)
Comment (rishky-msft) [reply ossdbtoolsservice/connection/connection_service.py]: resolved

File: ossdbtoolsservice/query_execution/query_execution_service.py (hunk 2 above)
Comment (rishky-msft) [reply ossdbtoolsservice/query_execution/query_execution_service.py:37]: resolved

**Review by nitishsharma25 — APPROVED** [review]

**Pull request merged**